rs/tests/common.bzl

```
@@ -5,18 +5,22 @@ Common dependencies for system-tests.
 load("@mainnet_icos_versions//:defs.bzl", "mainnet_icos_versions")
 load(":qualifying_nns_canisters.bzl", "QUALIFYING_NNS_CANISTERS", "QUALIFYING_SNS_CANISTERS")
 
-MAINNET_NNS_SUBNET_REVISION = mainnet_icos_versions["guestos"]["subnets"]["tdb26-jop6k-aogll-7ltgs-eruif-6kk7m-qpktf-gdiqx-mxtrf-vb5e6-eqe"]["version"]
-
-# TODO(NODE-1682): Switch mainnet_nns_setupos_disk_image to use dev image once published release is available.
-MAINNET_NNS_SUBNET_HASH = mainnet_icos_versions["guestos"]["subnets"]["tdb26-jop6k-aogll-7ltgs-eruif-6kk7m-qpktf-gdiqx-mxtrf-vb5e6-eqe"]["update_img_hash"]
-MAINNET_APPLICATION_SUBNET_REVISION = mainnet_icos_versions["guestos"]["subnets"]["io67a-2jmkw-zup3h-snbwi-g6a5n-rm5dn-b6png-lvdpl-nqnto-yih6l-gqe"]["version"]
-MAINNET_APPLICATION_SUBNET_HASH = mainnet_icos_versions["guestos"]["subnets"]["io67a-2jmkw-zup3h-snbwi-g6a5n-rm5dn-b6png-lvdpl-nqnto-yih6l-gqe"]["update_img_hash_dev"]
+MAINNET_LATEST_GUESTOS_REVISION = mainnet_icos_versions["guestos"]["latest_release"]["version"]
+MAINNET_LATEST_GUESTOS_HASH = mainnet_icos_versions["guestos"]["latest_release"]["update_img_hash"]
+MAINNET_LATEST_GUESTOS_DEV_HASH = mainnet_icos_versions["guestos"]["latest_release"]["dev_hash"]
```

optional nit:
```suggestion
MAINNET_LATEST_GUESTOS_UPDATE_HASH = mainnet_icos_versions["guestos"]["latest_release"]["update_img_hash"]
MAINNET_LATEST_GUESTOS_UPDATE_DEV_HASH = mainnet_icos_versions["guestos"]["latest_release"]["dev_hash"]
```

This would make the naming pretty long, but it is more precise. When I think of a "GuestOS hash," I would first think of the hash of the disk image, not the update image